features/step_definitions/admin_users_steps.rb

```diff
@@ -44,3 +44,24 @@
 Then(/^I expect the page to show me the number of admin users$/) do
   expect(page).to have_text("Admins (1)")
 end
+
+Then(/^I should see a column labeled "([^"]+)"$/) do |text|
+  find('th', text: text)
+end
+
+Given(/^there are various types of users in the system$/) do
+  @users = []
+
+  30.times do
```

Comment:
I think we can get the same spec result but much faster with just 2 user records